src/nxdoc/nuxeo-server/administration/elasticsearch-setup.md

```diff
@@ -516,31 +512,14 @@ In `/etc/default/elasticsearch` file you can increase the JVM heap to half of th
 ES_JAVA_OPTS="-Xms6g -Xmx6g"
 ```
 
-To prevent indexing errors like:
-
-```
-EsRejectedExceptionException[rejected execution (queue capacity 50)
-```
-
-Increase the bulk queue size In`/etc/elasticsearch/elasticsearch.yml` configuration file:
-
-```
-# For Elasticsearch 2.x
-# threadpool.bulk.queue_size: 500
-# For Elasticsearch 5.6
-# thread_pool.bulk.queue_size: 500
-# For Elasticsearch 6.x
-thread_pool.write.queue_size: 500
-```
-
-To reduce disk IO you should consider changing the default [translog](https://www.elastic.co/guide/en/elasticsearch/reference/6.3/index-modules-translog.html)
+To reduce disk IO you should consider changing the default [translog](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/index-modules-translog.html)
 durability from `request` to `async`.
 Since Nuxeo 10.3 this can be done from `nuxeo.conf`:
 ```
 elasticsearch.index.translog.durability=async
 ```
 
-If your indexes are already created you need some manual operation to change the translog:
+If your indices are already created you need some manual operation to change the translog:
```

Comment:
So far Nuxeo doc and code are using "indexes" as plural for index, 
it is true that elastic has switched to "indices", but using both makes things harder to grep
I would keep indexes unless there is a good reason.